<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceManager</class>
 <widget class="QMainWindow" name="ResourceManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="process">
     <attribute name="title">
      <string>进程管理</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>10</y>
        <width>611</width>
        <height>371</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>100</number>
      </attribute>
      <column>
       <property name="text">
        <string>PID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>状态</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>优先级</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>占用内存</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="killButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>387</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Kill</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>387</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="processnum">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>390</y>
        <width>361</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>进程数：</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="resource">
     <attribute name="title">
      <string>资源占用</string>
     </attribute>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>601</width>
        <height>101</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CPU</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="cpu_usage">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>521</width>
         <height>25</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>频率：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>70</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>最大睿频：</string>
       </property>
      </widget>
      <widget class="QLabel" name="cpu_frec">
       <property name="geometry">
        <rect>
         <x>92</x>
         <y>70</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="cpu_max">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>70</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_43">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>CPU：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_39">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>MHz</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>601</width>
        <height>191</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>内存和交换</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="mem_uasge">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>521</width>
         <height>25</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>70</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>大小：</string>
       </property>
      </widget>
      <widget class="QLabel" name="mem_used">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="mem_total">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>70</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_41">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>70</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>使用：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_42">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>内存：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_44">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>70</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>剩余：</string>
       </property>
      </widget>
      <widget class="QLabel" name="mem_left">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>70</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_46">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>160</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>剩余：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_47">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>交换：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_48">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>160</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>大小：</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="Swap_usage">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>130</y>
         <width>521</width>
         <height>25</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="Swap_total">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>160</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_50">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>使用：</string>
       </property>
      </widget>
      <widget class="QLabel" name="Swap_left">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>160</y>
         <width>141</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="Swap_used">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="shutdown_button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>340</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Shutdown</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reboot_button">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>340</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Reboot</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sysinfo">
     <attribute name="title">
      <string>系统信息</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>601</width>
        <height>61</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>用户信息</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>30</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>主机名：</string>
       </property>
      </widget>
      <widget class="QLabel" name="usrLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>usrLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="machineLabel">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>30</y>
         <width>251</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>machineLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>601</width>
        <height>131</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>处理器信息</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_38">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>99</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>MHz</string>
       </property>
      </widget>
      <widget class="QLabel" name="cpuLabel">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>41</y>
         <width>301</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>text</string>
       </property>
      </widget>
      <widget class="QLabel" name="cpuTypeLabel">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>70</y>
         <width>321</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>text</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>41</y>
         <width>38</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>CPU :</string>
       </property>
      </widget>
      <widget class="QLabel" name="freLabel">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>99</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>99</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>CPU频率 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>70</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>CPU类型 :</string>
       </property>
      </widget>
      <zorder>cpuLabel</zorder>
      <zorder>cpuTypeLabel</zorder>
      <zorder>label_16</zorder>
      <zorder>freLabel</zorder>
      <zorder>label_21</zorder>
      <zorder>label_19</zorder>
      <zorder>label_15</zorder>
      <zorder>label_38</zorder>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>222</y>
        <width>601</width>
        <height>181</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>系统信息</string>
       </property>
      </widget>
      <widget class="QLabel" name="sysVerLabel">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>41</y>
         <width>261</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>text</string>
       </property>
      </widget>
      <widget class="QLabel" name="syskernLabel">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>70</y>
         <width>281</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>text</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>41</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>系统版本  :</string>
       </property>
      </widget>
      <widget class="QLabel" name="sysmemLabel">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>99</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>text</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>99</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>内存大小 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_35">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>70</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>内核版本 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_36">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>127</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>启动时间 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="sysBegLabel">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>127</y>
         <width>341</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>text</string>
       </property>
      </widget>
      <widget class="QLabel" name="sysRunLabel">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>155</y>
         <width>341</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>text</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_54">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>155</y>
         <width>68</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>运行时间 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="sysmemLabel_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>100</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>GB</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="about">
     <property name="toolTipDuration">
      <number>0</number>
     </property>
     <attribute name="title">
      <string>关于</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>144</y>
        <width>57</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Abyssinica SIL</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>联系：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>30</y>
        <width>198</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Abyssinica SIL</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>程序版本：            </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>34</y>
        <width>122</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Abyssinica SIL</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manager V1.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>72</y>
        <width>160</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Abyssinica SIL</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>作者：            </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>77</y>
        <width>58</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Abyssinica SIL</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Haines</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>36</x>
        <y>108</y>
        <width>57</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Abyssinica SIL</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>时间：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>113</y>
        <width>84</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Abyssinica SIL</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2018.2.29</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>149</y>
        <width>187</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Abyssinica SIL</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>hainesluo@gmail.com</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
